init.lua

```diff
@@ -850,16 +850,16 @@ local fs_process_events
 -- Unique per-user to prevent players from making the counter wrap around for
 -- other players.
 local render_to_formspec_auto_name_ids = {}
--- if `embedded` is true, then this won't output a "root" formspec, but instead
---   will output  formspec designed to be embedded along with the information
---   that was removed from the form
-function Form:render_to_formspec_string(player, ctx, embedded)
+-- if `standalone` is not true, then this won't output a "root" formspec, but
+--   instead will output  formspec designed to be embedded along with the
+--   information that was removed from the form
```

Comment:
```suggestion
-- If standalone is set, this will return a standalone formspec, otherwise it
-- will return a formspec that can be embedded and a table with its size and
-- target formspec version
```

Reply:
Just applied suggestion.